### ISSUE TITLE
Group schema with closure inside Repeater does not pass $get

### ISSUE THREAD

**Issue by elasti-co** [laravel-filament/filament#5838]
### Package

filament/filament

### Package Version

^v2.17.9

### Laravel Version

v10.1

### Livewire Version

v2.0

### PHP Version

PHP 8.2

### Problem description

Group schema inside Repeater when filled with closure does not pass \Closure $get attributes. Instead it returns NULL.

### Expected behavior

Like in 2.17.8 and before, it had passed $get parameters of the form.

### Steps to reproduce

``` php
Forms\Components\Repeater::make('users')
    ->schema([
        Forms\Components\TextInput::make('name')->reactive(),
        Forms\Components\Group::make()
            ->schema(function (\Closure $get) {
                // TODO Fix $get (now it returns NULL)
                return [
                    Forms\Components\TextInput::make('password')
                        ->visible($get('name') === 'John'),
                ];
            }),
    ]),
```

### Reproduction repository

https://github.com/elasti-co/filament_bug

**Comment by danharrin** [comment]
Please update your Filament version as I think we fixed this

**Comment by elasti-co** [comment]
> Please update your Filament version as I think we fixed this

Dan, thank you for filament and reply ) 
Checked version 2.17.10. Still not working. Maybe 2.17.11 will do, but it's not yet available in composer.

**Comment by danharrin** [comment]
Ah that's weird, looks like our GitHub actions failed to release. It should be available in Composer now

**Comment by elasti-co** [comment]
Yeah, now it's available in Composer. Updated.

But still face the problem...
The latest version it worked was 2.17.8.

**Comment by danharrin** [comment]
Fixed by #5845, let me know if not

**Comment by elasti-co** [comment]
@danharrin Thank you, now it works 🤩

**Comment by MaciejLunitz** [comment]
Laravel 10.48, Filament 3.2, January 2025
I still couldn't pass $get in the Repeater schema until I found this issue yesterday.
Thanks to @elasti-co and his top post I have tried to put my schema inside the Group. And there $get is passing through the function.

I haven't found any other discussion or docs for this case. Even Copilot suggests wrong way (directly in schema).

@danharrin, should it work (to pass $get) directly in Repeater schema, or not?
